playbooks/nginxplus_production.yml

```diff
@@ -1,17 +1,16 @@
 ---
-- hosts: nginxplus
+- name: update nginx
+  hosts: nginxplus
   remote_user: pulsys
-  strategy: free
   become: true
 
+# updates existing load balancer
   roles:
-    - role: ../roles/datadog
-    - role: ../roles/deploy_user
     - role: ../roles/nginxplus
 
-- hosts: nginxplus
+- name: restart nginx and announce success
+  hosts: nginxplus
   remote_user: pulsys
-  strategy: linear
```

Comment:
I think we still want the linear strategy on the restart so that both load balancer boxes don't go down for restart at the same time, which results in a downtime blip for all our sites.

Reply:
Linear is the [default strategy](https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html). We can reinstate the line for clarity, though.

Reply:
Oh, interesting I didn't know that.

well if we're running it all as linear than I guess it's not needed. Feel free to remove it from both if you like and I'll go ahead and approve so I'm not a blocker either way.